### PR TITLE
minor: added who has access to source code/device images

### PR DESCRIPTION
**Pull request by NatasaKyvetou**
Added some description about access to device, source code and images to the `Device access` section after checking with the devices team that the wording is clear and accurate.

---

> *Please make sure to read the [CONTRIBUTING](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) document before opening the PR for relevant information on contributing to the documentation. Thanks!*

### CODE REVIEW

File: pages/learn/welcome/security.md [hunk 1]
```diff
@@ -25,7 +25,9 @@ The first points of access to the {{ $names.company.lower }} ecosystem are the u
...
+Device access is managed by our host operating system, [{{ $names.os.lower }}](/reference/OS/overview/2.x). {{ $names.os.upper }} is a thin Linux environment that supports the {{ $names.company.lower }} services and user application containers.  {{ $names.os.upper }} is built using [Yocto Linux](https://www.yoctoproject.org/), the de facto standard for building lightweight embedded Linux environments. Using Yocto allows {{ $names.company.lower }} to build images that contain no unused or unnecessary code in either userspace or the running kernel, minimizing the device's available attack surface. All {{ $names.company.lower }} software running on devices is 100% open source and can be independently audited and verified. Device access is granted to a subset of {{ $names.company.lower }} employees to enable support and device troubleshooting. This access is controlled by ssh key access and only after access is granted to balena by the device fleet manager.
```
Comment (vipulgupta2048) [inline pages/learn/welcome/security.md:28]:
```suggestion
Device access is managed by our host operating system, [{{ $names.os.lower }}](/reference/OS/overview/2.x). {{ $names.os.upper }} is a thin Linux environment that supports the {{ $names.company.lower }} services and user application containers. {{ $names.os.upper }} is built using [Yocto Linux](https://www.yoctoproject.org/), the de facto standard for building lightweight embedded Linux environments. Using Yocto allows {{ $names.company.lower }} to build images that contain no unused or unnecessary code in either userspace or the running kernel, minimizing the device's available attack surface. All {{ $names.company.lower }} software running on devices is 100% open source and can be independently audited and verified. Device access is granted to a subset of {{ $names.company.lower }} employees to enable support and device troubleshooting. This access is controlled by ssh key access and only after access is granted to balena by the device fleet manager.
```

@NatasaKyvetou The device fleet manager is not an actual role and we might need to either omit it or specify explicitly who can provide support access.

Comment (vipulgupta2048) [reply pages/learn/welcome/security.md:28]: It's not super clear what change I suggested above. It's an extra space you added in the start of the third sentence

Comment (NatasaKyvetou) [reply pages/learn/welcome/security.md]: @vipulgupta2048 I understand this as whoever manages the fleet, so the user but I get what you mean as someone might read it and wonder. Does it make sense to completely remove it and stop the sentence to '...granted to balena'?

**Comment by NatasaKyvetou** [comment]
@balena-ci rebase

**Comment by NatasaKyvetou** [comment]
@balena-ci I self-certify!

**Comment by vipulgupta2048** [comment]
Awesome work @NatasaKyvetou

**Pull request merged**